docs/src/main/asciidoc/kafka-dev-services.adoc

```diff
@@ -27,14 +27,19 @@ You can configure the broker address using `kafka.bootstrap.servers`.
 
 == Setting the port
 
-By default, Dev Services for Kafka picks a random port and configure the application.
+By default, Dev Services for Kafka picks a random port and configures the application.
 You can set the port by configuring the `quarkus.kafka.devservices.port` property.
 
 Note that the Kafka advertised address is automatically configured with the chosen port.
 
 == Configuring the image
 
-By default, Dev Services for Kafka uses: `vectorized/redpanda:latest`.
-You can select any version from https://hub.docker.com/r/vectorized/redpanda.
+Dev Services for Kafka uses: `vectorized/redpanda` images.
+You can select any version from https://hub.docker.com/r/vectorized/redpanda:
+
+[source, properties]
+----
+quarkus.kafka.devservices.image-name=vectorized/redpanda:latest
+----
```

Comment:
Thanks! I improved this section of this file in my Apicurio Registry PR as well, but this is even better. Will steal it from you :-)